src/OpenTelemetry.AutoInstrumentation/Configurations/LogSettings.cs

```diff
@@ -18,9 +18,9 @@ internal class LogSettings : Settings
     public bool LogsEnabled { get; private set; }
 
     /// <summary>
-    /// Gets the logs exporter.
+    /// Gets the list of enabled logs exporters.
```

Comment:
```suggestion
    /// List of logs exporters to be used.
```

based on https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/fc1af32094d792221d63ad5b9f31f453d73e671f/src/OpenTelemetry.AutoInstrumentation/Configurations/SdkSettings.cs#L17

Reply:
I think that the `Gets` predix is enforced by some code analysis. Worth to double check before commiting.